.ci/table_of_content.py

```diff
@@ -51,6 +50,19 @@ def get_tc_line(title, title_for_tc, link):
     return line
 
 
+def is_ref_to_top_exists(cell, idx):
+    ref_exists = False
+    for row in cell[idx + 1:]:
+        row = row.strip()
+        if '[back to top ⬆️](#Table-of-content:)' in row:
```

Comment:
Should be Table of contents (with s):
https://english.stackexchange.com/questions/207086/table-of-contents-vs-table-of-content

Reply:
done